microsoft_student_partners/hackcovid19/organizers.json

```diff
@@ -14,5 +14,14 @@
     "twitter": "https://twitter.com/_adityaraman",
     "instagram": "",
     "position": "Core Organizer"
+  }, 
+  "Aashrut Vaghani": {
+    "name": "Aashrut Vaghani",
+    "image_url": "https://drive.google.com/file/d/1fonMoJLmd21aeTC2u6VWHUHEOR9uirbe/view?usp=drivesdk",
```

Comment:
Drive link doesn't work please share your image link like url of profile pic of Linkedin or github